src/app/pages/TokenDashboardPage/hook.ts

```diff
@@ -72,7 +72,8 @@ export const _useTokenTransfers = (scope: SearchScope, params: undefined | GetRu
     network,
     layer, // This is OK since consensus has been handled separately
     {
-      ...pagination.paramsForQuery,
```

Comment:
paramsForQuery is unused now

Reply:
This change was a stowaway from #1358. Removed from there.